packages/create-powerbi-visual/README.md

```diff
@@ -0,0 +1,4 @@
+# Project Creator for Essex PowerBI Visuals
+
+## Usage
+`create-powerbi-visual <name>`
```

Comment:
Is there any other configuration needed for this that would be nice in the docs?

Reply:
Nope

Reply:
What may make sense is to update the new visual file's readme with some usage notes on how to use the new package.